ReactWindows/ReactNative.Shared/UIManager/LayoutAnimation/StoryboardObservable.cs

```diff
@@ -26,8 +30,13 @@ public IDisposable Subscribe(IObserver<Unit> observer)
             _storyboard.Begin();
 
             return Observable.FromEventPattern<object>(
+#if WINDOWS_UWP
                 h => _storyboard.Completed += h,
                 h => _storyboard.Completed -= h)
+#else
+                h => _storyboard.Completed += new EventHandler(h),
+                h => _storyboard.Completed -= new EventHandler(h))
+#endif
```

Comment:
Can you do...
#if WINDOWS_UWP
            return Observable.FromEventPattern<object>(
                h => _storyboard.Completed += h,
                h => _storyboard.Completed -= h)
#else
            return Observable.FromEventPattern(
                h => _storyboard.Completed += h,
                h => _storyboard.Completed -= h)
#endif